game/Structs.h

```diff
@@ -45,6 +45,9 @@ struct OP2 MAP_RECT
 	void ClipToMap();
 	void FromPtSize(const LOCATION&, const LOCATION&);
 	int Height() const;
+	// Use positive values to shrink the MAP_RECT and negative values to grow the MAP_RECT
+	//     unitsWide: X1 increases by value, X2 decreases by value
+	//     unitsHigh: Y1 increases by value, Y2 decreases by value
 	void Inflate(int unitsWide, int unitsHigh);
```

Comment:
That does sound confusing.

I think the more common operation is `inset`, which shrinks a rect for positive values. This operation is named to imply the reverse.